ci/build.yml

```diff
@@ -439,7 +439,7 @@ jobs:
                 setvar release_sha "$(added_line | awk '{print $1}')"
                 RELEASE_TAG="$(added_line | awk '{print $2}')"
                 setvar release_tag "$RELEASE_TAG"
-                if [ $(added_line | awk '{print $3}') == "SPLIT_RELEASE" ]; then
+                if [ "$(added_line | awk '{print $3}')" == "SPLIT_RELEASE" ]; then
```

Comment:
Without the quotes, when the release is not a split release, this expands to
```
[  == "SPLIT_RELEASE" ]
```
which crashes with the error message:
````
[: ==: unary operator expected
```
because `==` is given a single argument instead of the 2 it expects. With the quotes, it does get two arguments, the first one of which is the empty string.